tests/new_functional/spec/census-household.spec.js

```diff
@@ -0,0 +1,331 @@
+const Volunteering = require('../pages/surveys/census/household/volunteering.page');
+const EmploymentType = require('../pages/surveys/census/household/employment-type.page');
+const Jobseeker = require('../pages/surveys/census/household/jobseeker.page');
+const JobAvailability = require('../pages/surveys/census/household/job-availability.page');
+const JobPending = require('../pages/surveys/census/household/job-pending.page');
+const Occupation = require('../pages/surveys/census/household/occupation.page');
+const EverWorked = require('../pages/surveys/census/household/ever-worked.page');
+const MainJob = require('../pages/surveys/census/household/main-job.page');
+const JobTitle = require('../pages/surveys/census/household/job-title.page');
+const JobDescription = require('../pages/surveys/census/household/job-description.page');
+const EmployersBusiness = require('../pages/surveys/census/household/employers-business.page');
+const MainJobType = require('../pages/surveys/census/household/main-job-type.page');
+const BusinessName = require('../pages/surveys/census/household/business-name.page');
+const HouseholdMemberCompleted = require('../pages/surveys/census/household/household-member-completed.page');
+const VisitorBegin = require('../pages/surveys/census/household/visitor-begin.page');
+const VisitorName = require('../pages/surveys/census/household/visitor-name.page');
+const VisitorSex = require('../pages/surveys/census/household/visitor-sex.page');
+const VisitorDateOfBirth = require('../pages/surveys/census/household/visitor-date-of-birth.page');
+const VisitorUkResident = require('../pages/surveys/census/household/visitor-uk-resident.page');
+const VisitorAddress = require('../pages/surveys/census/household/visitor-address.page');
+const VisitorCompleted = require('../pages/surveys/census/household/visitor-completed.page');
+const VisitorsCompleted = require('../pages/surveys/census/household/visitors-completed.page');
+const Confirmation = require('../pages/confirmation.page');
+const ThankYou = require('../pages/thank-you.page');
+
+
+describe('Census Household', function () {
+
+  it('Given a census household survey with welsh region, When i enter valid data, Then the survey should submit successfully', function () {
+      return helpers.startCensusQuestionnaire('census_household.json', false, 'GB-WLS').then(() => {
+        return browser
+        // who-lives-here
+          .click(PermanentOrFamilyHome.yes())
+          .click(PermanentOrFamilyHome.submit())
+          .setValue(HouseholdComposition.firstName(0), 'John')
+          .click(HouseholdComposition.submit())
+          .click(EveryoneAtAddressConfirmation.yes())
+          .click(EveryoneAtAddressConfirmation.submit())
+          .setValue(OvernightVisitors.answer(), 1)
+          .click(OvernightVisitors.submit())
+          .click(WhoLivesHereCompleted.submit());
+
+      }).then(completeHouseholdAndAccommodation).then(() => {
```

Comment:
i think he 2nd then should be on a new line?
when i first looked over this it wasn't immediately obvious that something was happening between the 2 blocks